osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/DlmsHelperService.java

```diff
@@ -684,7 +700,7 @@ private void appendItemValues(final DataObject dataObject, final StringBuilder b
     }
 
     private static String getDataType(final DataObject dataObject) {
```

Comment:
Since this class is seriously too large I would suggest hiding this (ugly) mapping method away in a separate class with a static method. (I realize you didn't really touch this method so consider if you want to boy scout this or not)

Reply:
I suggest to leave it for now as it is.